wgpu-core/src/instance.rs

```diff
@@ -195,6 +195,26 @@ impl Surface {
 
         Ok(caps)
     }
+
+    pub fn get_default_configure<A: HalApi>(
```

Comment:
Can't we implement this completely in user space by using the other functions? This doesn't feel like it needs any special magic.

Reply:
You're right, after leaving `get_default_configure` only in the wgpu crate, the code is now simpler.